conf/openlibrary.yml

```diff
@@ -162,6 +162,9 @@ ia_ol_xauth_s3:
     s3_key: XXX
     s3_secret: XXX
 
+bwb_api:
+    url: http://products.betterworldbooks.com/service.aspx?ItemId=
```

Comment:
I was going to check whether this should be https -- looks like the endpoint only responds via http though.